store/src/java/com/zimbra/cs/util/ProxyConfGen.java

```diff
@@ -2407,7 +2407,9 @@ private static void fillVarsWithDomainAttrs(DomainAttrItem item)
                 vip = InetAddress.getByName(item.virtualIPAddress);
             }
         } catch (UnknownHostException e) {
-            throw new ProxyConfException("virtual host name \"" + item.virtualHostname + "\" is not resolvable", e);
+            mLog.warn("virtual host name \"" + item.virtualHostname + "\" is not resolvable");
```

Comment:
A minor nit (but one that Gren has pointed out to me before).. Instead of appending multiple strings to compose the message, use something like this:

     String.format("virtual host name \"%s\" is not resolvable", item.virtualHostname)

Reply:
Will make this change.